pkgs/build-support/node/build-npm-package/default.nix

```diff
@@ -37,12 +37,45 @@ let
     hash = npmDepsHash;
   };
 
+  patchNode =
```

Comment:
I'm really not sure how to properly expose this for individual usage with the setup hooks... like, `buildNpmPackage.patchNode`?